src/main/resources/changelogs/01-flights/06-event-definitions.sql

```diff
@@ -0,0 +1,60 @@
+--liquibase formatted sql
```

Comment:
Sinec this only gets ran once when setting up, why not break up the insert statement by event definition type for readability? i.e group Total Fuel together as one insert statement

Reply:
It would probably be good to do. I originally just copy and pasted this from the event definition blobs.

Reply:
we need to make sure these are the most up to date ones from the database (not some other location). i did go in to fix a couple bugs with a few of these not too long ago (related to CHT and EGT events).